test/integration/concierge_tls_spec_test.go

```diff
@@ -0,0 +1,221 @@
+package integration
```

Comment:
For this project, all brand new Go files should have our copyright statement that includes only the current year, e.g. `2024`. All updated Go files should have their copyright dates extended to include the year of file creation and the year of most recent edit, e.g. `2021-2024`.

There are scripts to help with this. Please enable the pre-commit hooks which will help remind about this. https://github.com/vmware-tanzu/pinniped/blob/main/CONTRIBUTING.md#pre-commit-hooks